game-headed/src/main/resources/org/triplea/game/client/ui/javafx/fxml/RoleSelectionScreen.fxml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.geometry.*?>
+<?import javafx.scene.control.*?>
+<?import javafx.scene.layout.*?>
+
+<VBox fx:id="root" alignment="TOP_CENTER" fillWidth="false" spacing="5.0" xmlns="http://javafx.com/javafx/10.0.2-internal" xmlns:fx="http://javafx.com/fxml/1" fx:controller="org.triplea.game.client.ui.javafx.screens.RoleSelection">
+   <children>
+      <HBox alignment="CENTER">
+         <children>
+            <Label text="Battle Orders" />
+         </children>
+      </HBox>
+      <ScrollPane>
+         <content>
+            <GridPane fx:id="factionGrid" hgap="5.0" vgap="5.0">
+              <columnConstraints>
+                <ColumnConstraints hgrow="SOMETIMES" />
+                <ColumnConstraints hgrow="SOMETIMES" />
+                  <ColumnConstraints hgrow="SOMETIMES" />
```

Comment:
Any idea if xmllint or spotless works on fxml files?
If so, any objections to using either?

Reply:
This is a side-note for consideration, not to be done in this PR.

Reply:
Well FXML should be 100% valid XML AFAIK, but I'm using SceneBuilder for the UI (a WYSIWYG editor for the most part), so all of the XML is auto-generated anyways, so not much value in style checking that IMO.

Reply:
`WYSIWYG editor` for doing non-prototype work always makes me nervous : |

Anyways, when you get a chance, so others can jump in, please add some dev setup notes on which editor you are using and kindly provide a link to install it.

Reply:
Will do, it's pretty straightforward and IntelliJ integrates with it well.

I'd highly recommend trying it out. It makes dealing with details (like padding or visual effects like shadows etc.) easy to work with.
The main reason to use it is because there's no way to remember all of the available features, so having a couple of checkboxes to check and uncheck to see if it changes something comes in really handy most of the time.

Obviously it doesn't solve all problems, but it should allow non-coders to work on the UI if they want to bring in their own ideas

Reply:
Obviously it's only useful for static content and to quickly prototype dynamic content (when you're not getting why it doesn't look like it's supposed to look)